lib/ex_unit/lib/ex_unit/filters.ex

```diff
@@ -14,51 +16,63 @@ defmodule ExUnit.Filters do
   on the command line) includes a line number filter, and if so returns the
   appropriate ExUnit configuration options.
   """
-  @spec parse_path(String.t()) :: {String.t(), Keyword.t()}
-  def parse_path(file) do
-    case extract_line_numbers(file) do
-      {path, []} -> {path, []}
-      {path, line_numbers} -> {path, exclude: [:test], include: line_numbers}
-    end
+  @spec parse_path(String.t()) :: {String.t(), ex_unit_opts}
+  def parse_path(file_path) do
+    {[parsed_path], ex_unit_opts} = parse_paths([file_path])
+    {parsed_path, ex_unit_opts}
```

Comment:
The `parse_path/1` function seems to be obsolete now. It's only used in the tests. Maybe we should remove it?

On a related note, I was thinking about the case when only one file with line numbers is given on the CLI. I think for shorter outputs we should return `{:line, 123}` as a filter option instead of `{:location, {file, 123}}`.